fabric-transitive-access-wideners-v1/src/main/resources/fabric-transitive-access-wideners-v1.accesswidener

```diff
@@ -53,6 +53,10 @@ transitive-accessible method net/minecraft/block/Material$Builder burnable ()Lne
 transitive-accessible method net/minecraft/block/Material$Builder destroyedByPiston ()Lnet/minecraft/block/Material$Builder;
 transitive-accessible method net/minecraft/block/Material$Builder blocksPistons ()Lnet/minecraft/block/Material$Builder;
 
+# Custom Sign Types
+transitive-accessible method net/minecraft/util/SignType <init> (Ljava/lang/String;)V
```

Comment:
We don't usually access-widen ones with `String` argument (only `Identifier`) - but I think making a new class just for this doesn't sound good.